committeeoversightapp/models.py

```diff
@@ -51,6 +51,19 @@ def latest_rating(self):
         rating_set = self.committeerating_set.order_by('-congress')
         return rating_set[0]
 
+
+    def sliced_name(self):
```

Comment:
Something like `display_name` would be a little more straightforward. It might also be nice to define this as a property (adding the `@property` decorator), since it's more of a derived attribute of an instance than functionality in and of itself! What do you think?

Reply:
I think we're going to be using the full (unsliced) name for the committee detail pages, and this condensed name is just for tables where space is precious. Is there another name that makes more sense to you, with that in mind? Maybe `short_name`? I'll add the property decorator.

Reply:
`short_name` sounds great!